src/unit/engine/UberAndroidTestEngine.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+final class UberAndroidTestEngine extends ArcanistUnitTestEngine {
```

Comment:
lol.... looks like you're right. I just copied it from the project it was in and didn't check its contents.

let me reach out to the author since it looks like it also relies on a local file to a project (`gradlew`).